tests/baselines/reference/classUpdateTests.js

```diff
@@ -190,7 +190,7 @@ var G = /** @class */ (function (_super) {
 }(D));
 var H = /** @class */ (function () {
     function H() {
-        _this = _super.call(this) || this;
+        return _super.call(this) || this;

```

Comment:
Note that this is a behavioral change for when `H` is used incorrectly as a callable function:

```
let invalid = H();
```

...but the new output is a correction, as when `H` is changed to `extend Object`,  the `return` does exist.